source/Octopus.Tentacle.Client/Scripts/ScriptExecutionOrchestrator.cs

```diff
@@ -68,19 +73,21 @@ public async Task<ScriptStatusResponseV2> ExecuteScript(CancellationToken script
             {
                 try
                 {
-                    scriptStatusResponse = await rpcCallRetryHandler.ExecuteWithRetries(
+                    scriptStatusResponse = await rpcCallExecutor.ExecuteWithRetries(
+                        nameof(scriptServiceV2.StartScript),
```

Comment:
What does the rpcCallName end up using nameof here? Does it include the service and operation name?

Reply:
No, it will just be `StartScript`.

Reply:
Probably not needed now but if we log RPC Metrics in Server it may be necessary to know which service the operation is on e.g. SSv1 and SSv2 both have a StartScript operation

Reply:
Agreed.

I might keep it simple and leave it as is for now until we need it.